### PR TITLE
Prepare for first release

### PR DESCRIPTION
**Pull request by Leechael**

### CODE REVIEW

File: src/lib/runQuickJs.ts [hunk 1]
```diff
@@ -1,9 +1,123 @@
...
+    options.body = hexToString(options.body)
+  }
+  const res = request(options.method || 'GET', options.url, {
+    headers: options.headers,
+    body: options.body,
+  })
+  return {
+    statusCode: res.statusCode,
+    body: res.getBody(options.returnTextBody ? 'utf8' : '')
+  }
```
Comment (Leechael) [inline src/lib/runQuickJs.ts:32]: We need test for the fail cases and check is it any potential exceptions not yet handle.

File: src/lib/runQuickJs.ts [hunk 2]
```diff
@@ -1,9 +1,123 @@
...
+}
+
+function syncRequest(options: {
+  url: string,
+  method?: HttpVerb,
+  headers?: Record<string, string>,
+  body?: string,
+  returnTextBody?: boolean,
+}) {
+  if (options.body && isHexString(options.body)) {
```
Comment (Leechael) [inline src/lib/runQuickJs.ts]: Add `typeof` check.

File: src/lib/runQuickJs.ts [hunk 3]
```diff
@@ -1,9 +1,123 @@
-import { getQuickJS } from 'quickjs-emscripten'
+import { getQuickJS, QuickJSContext } from 'quickjs-emscripten'
+
+import { HttpVerb} from 'then-request'
```
Comment (Leechael) [inline src/lib/runQuickJs.ts]: `import { type HttpVerb } from 'then-request'`

File: src/lib/runQuickJs.ts (hunk 1 above)
Comment (pacoyang) [reply src/lib/runQuickJs.ts:32]: It will throw error messages to the console by now.

Comment (Leechael) [reply src/lib/runQuickJs.ts:32]: We should ensure it's behavior same in real environment.

File: src/commands/init.ts [hunk 4]
```diff
@@ -7,13 +7,9 @@ import inquirer from 'inquirer'
...
-  'lens-oracle': {
-    url: 'https://github.com/pacoyang/lens-stats',
-    description: 'A minimal oracle template for lens.',
-  },
-  'phat-oracle': {
-    url: 'https://github.com/pacoyang/phat-oracle-contract',
-    description: 'A phat oracle contract template.',
+  'lensapi-oracle-consumer-contract': {
+    url: 'https://github.com/Phala-Network/lensapi-oracle-consumer-contract',
+    description: 'Polygon Consumer Contract for LensAPI Oracle',
```
Comment (Leechael) [inline src/commands/init.ts:12]: Pls update README.md as well.

File: src/commands/watch.ts [hunk 5]
```diff
@@ -83,6 +91,9 @@ export default class Watch extends Command {
         // actions
         [action],
       )
+      if (flags.onece) {
```
Comment (Leechael) [inline src/commands/watch.ts]: typo

File: src/commands/watch.ts [hunk 6]
```diff
@@ -17,6 +17,16 @@ export default class Watch extends Command {
     rpc: Flags.string({
       description: 'RPC endpoint',
     }),
+    scriptArgs: Flags.string({
+      char: 'a',
+      description: 'Script Arguments',
+      multiple: true,
+      default: [],
+    }),
+    onece: Flags.boolean({
```
Comment (Leechael) [inline src/commands/watch.ts]: typo

**Pull request merged**